microsoft/testsuites/hpc/infinibandsuite.py

```diff
@@ -100,12 +100,16 @@ def verify_hpc_over_sriov(self, log: Logger, node: Node) -> None:
             "ib_uverbs",
             "ib_core",
             "mlx5_core",
-            "mlx_compat",
             "rdma_cm",
             "iw_cm",
             "ib_cm",
         ]
 
+        # mlx_compat is only needed if using Mellanox OFED. CBLMariner uses
+        # the inbox Mellanox core driver provided by kernel which does not need this
+        if not isinstance(node.os, CBLMariner):
+            expected_modules.append("mlx_compat")
```

Comment:
Please rebase code, https://github.com/microsoft/lisa/pull/3134 has removed this module.